WordPress/Classes/Models/WPAccount.m

```diff
@@ -128,4 +128,21 @@ - (BOOL)isWPComAccount
     return self.restApi != nil;
 }
 
+#pragma mark - Sorting support
+
+- (NSComparisonResult)compare:(WPAccount *)account
+{
+    BOOL account1IsWPComAccount = [self isWPComAccount];
+    BOOL account2IsWPComAccount = [account isWPComAccount];
```

Comment:
Well, I think this method can safely return `NSOrderedSame` for all cases then.

Regarding the sorting, it's taken care of by the existing logic (which compares the default account across blogs) so no other changes would be necessary.

Right?